core/include/dt-bindings/firewall/stm32mp25-rifsc.h

```diff
@@ -0,0 +1,173 @@
+#define STM32MP25_RIFSC_SDMMC1_ID		76
+#define STM32MP25_RIFSC_SDMMC2_ID		77
+#define STM32MP25_RIFSC_SDMMC3_ID		78
+#define STM32MP25_RIFSC_GPU_ID			79
+#define STM32MP25_RIFSC_LTDC_CMN_ID		80
+#define STM32MP25_RIFSC_DSI_CMN_ID		81
+#define STM32MP25_RIFSC_LVDS_ID			84
+#define STM32MP25_RIFSC_CSI_ID			86
+#define STM32MP25_RIFSC_DCMIPP_ID		87
+#define STM32MP25_RIFSC_DCMI_PSSI_ID		88
+#define STM32MP25_RIFSC_VDEC_ID			89
+#define STM32MP25_RIFSC_VENC_ID			90
+#define STM32MP25_RIFSC_RNG_ID			92
+#define STM32MP25_RIFSC_PKA_ID			93
+#define STM32MP25_RIFSC_SAES_ID			94
+#define STM32MP25_RIFSC_HASH_ID			95
+#define STM32MP25_RIFSC_CRYP1_ID		96
+#define STM32MP25_RIFSC_CRYP2_ID		97
+#define STM32MP25_RIFSC_IWDG1_ID		98
+#define STM32MP25_RIFSC_IWDG2_ID		99
+#define STM32MP25_RIFSC_IWDG3_ID		100
+#define STM32MP25_RIFSC_IWDG4_ID		101
+#define STM32MP25_RIFSC_IWDG5_ID		102
+#define STM32MP25_RIFSC_WWDG1_ID		103
+#define STM32MP25_RIFSC_WWDG2_ID		104
+#define STM32MP25_RIFSC_VREFBUF_ID		106
+#define STM32MP25_RIFSC_DTS_ID			107
+#define STM32MP25_RIFSC_RAMCFG_ID		108
+#define STM32MP25_RIFSC_CRC_ID			109
+#define STM32MP25_RIFSC_SERC_ID			110
+#define STM32MP25_RIFSC_OCTOSPIM_ID		111
+#define STM32MP25_RIFSC_GICV2M_ID		112
+#define STM32MP25_RIFSC_I3C1_ID			114
+#define STM32MP25_RIFSC_I3C2_ID			115
+#define STM32MP25_RIFSC_I3C3_ID			116
+#define STM32MP25_RIFSC_I3C4_ID			117
+#define STM32MP25_RIFSC_ICACHE_DCACHE_ID	118
+#define STM32MP25_RIFSC_LTDC_L0L1_ID		119
+#define STM32MP25_RIFSC_LTDC_L2_ID		120
+#define STM32MP25_RIFSC_LTDC_ROT_ID		121
+#define STM32MP25_RIFSC_DSI_TRIG_ID		122
+#define STM32MP25_RIFSC_DSI_RDFIFO_ID		123
+#define STM32MP25_RIFSC_OTFDEC1_ID		125
+#define STM32MP25_RIFSC_OTFDEC2_ID		126
+#define STM32MP25_RIFSC_IAC_ID			127
+
+#define RIFSC_RISC_CFEN_SHIFT			0
+#define RIFSC_RISC_SEM_EN_SHIFT			1
+#define RIFSC_RISC_SCID_SHIFT			4
+#define RIFSC_RISC_SEC_SHIFT			8
+#define RIFSC_RISC_PRIV_SHIFT			9
+#define RIFSC_RISC_LOCK_SHIFT			10
+#define RIFSC_RISC_SEML_SHIFT			16
+#define RIFSC_RISC_PER_ID_SHIFT			24
+#define RIFSC_RISC_PERx_CID_SHIFT		0
+
+/* Global lock bindings */
+#define RIFSC_RIMU_GLOCK			1
+#define RIFSC_RISUP_GLOCK			2
+
+#define RIFPROT(rifid, sem_list, lock, sec, priv, scid, sem_en, cfen) \
+	(((rifid) << RIFSC_RISC_PER_ID_SHIFT) | \
+	 ((sem_list) << RIFSC_RISC_SEML_SHIFT) | \
+	 ((lock) << RIFSC_RISC_LOCK_SHIFT) | \
+	 ((priv) << RIFSC_RISC_PRIV_SHIFT) | \
+	 ((sec) << RIFSC_RISC_SEC_SHIFT) | \
+	 ((scid) << RIFSC_RISC_SCID_SHIFT) | \
+	 ((sem_en) << RIFSC_RISC_SEM_EN_SHIFT) | \
+	 ((cfen) << RIFSC_RISC_CFEN_SHIFT))
+
+/* masters ID */
+#define RIMU_ID(idx)		(idx)
+
+/* master configuration modes */
+#define RIF_CIDSEL_P	0x0 /* config from RISUP */
+#define RIF_CIDSEL_M	0x1 /* config from RIMU */
+
+#define RIFSC_RIMC_MODE_SHIFT		2
+#define RIFSC_RIMC_MCID_SHIFT		4
+#define RIFSC_RIMC_MSEC_SHIFT		8
+#define RIFSC_RIMC_MPRIV_SHIFT		9
+#define RIFSC_RIMC_M_ID_SHIFT		16
+#define RIFSC_RIMC_ATTRx_SHIFT		0
```

Comment:
This macro is not used, better remove its definition.

Reply:
It is used lign 275 through _RIF_FLD_GET()

Reply:
Right! my bad. I could not find it grepping the code but I forgot you defined these macros.